src/containers/Token/TokenTransactionTable/test/TokenTransactionTable.test.tsx

```diff
@@ -20,13 +21,11 @@ const TEST_CURRENCY = 'abc'
 
 describe('TokenTransactionsTable container', () => {
   const createWrapper = (
-    getAccountTransactionsImpl = () =>
-      new Promise(
-        () => {},
-        () => {},
-      ),
+    getAccountTransactionsImpl = () => new Promise(() => {}),
   ) => {
-    getAccountTransactions.mockImplementation(getAccountTransactionsImpl)
+    ;(getAccountTransactions as Mock).mockImplementation(
```

Comment:
Why as

Reply:
Why leading semi

Reply:
So that typescript would not complain about calling mockImplementation.  Typsecript doesnt always know when you have mocked something with jest